src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <Age age={person.age} />
+    <Partner
+      isMarried={person.isMarried}
+      sex={person.sex}
+      partnerName={person.partnerName}
+    />
+  </section>
+);
+
```

Comment:
we usually use separate files for each component. 
So there should be `Age.tsx` and `Partner.tsx`, which are exported from their own files. I'd suggest you do it too